src/main/resources/db/competition.csv

```diff
@@ -0,0 +1,34 @@
+Name;Description
+Competition1;test description of competition1
```

Comment:
It's probably not required to have more than 2 initially configured competitions. It's actually enough to have 2 for local testing purposes.

Reply:
3 left only.